src/Microsoft.DotNet.Cli.Utils/CommandResolver.cs

```diff
@@ -181,8 +181,7 @@ private static CommandSpec ConfigureCommandFromPackage(string commandName, IEnum
 
                 if (depsPath != null)
                 {
-                    additionalArgs.Add("--depsfile");
-                    additionalArgs.Add(depsPath);
+                    additionalArgs.Add($"--depsfile:{depsPath}");
```

Comment:
Host arg parsing logic is pretty simple as we are dealing with c++. It expects this input As a single string so it doesn't need to reason about multiple elements of argv. Our escaping will be right since it will quote the entire string.